pkg/util/util.go

```diff
@@ -113,6 +113,14 @@ func (intstr *IntOrString) UnmarshalJSON(value []byte) error {
 	return json.Unmarshal(value, &intstr.IntVal)
 }
 
+// AsString returns the string value, or Itoa's the int value.
+func (intstr *IntOrString) AsString() string {
```

Comment:
I can change it-- I did AsString because I figured the omission must be on purpose...